charts/beyla/README.md

```diff
@@ -53,26 +53,32 @@ eBPF-based autoinstrumentation HTTP, HTTP2 and gRPC services, as well as network
 | resources | object | `{}` |  |
 | securityContext | object | `{"privileged":true}` | Security context for privileged setup. |
 | service.annotations | object | `{}` | Service annotations. |
-| service.appProtocol | string | `""` | Adds the appProtocol field to the service. This allows to work with istio protocol selection. Ex: "http" or "tcp" |
 | service.clusterIP | string | `""` | cluster IP |
-| service.enabled | bool | `false` | whether to create a service for internal metrics |
+| service.enabled | bool | `false` | whether to create a service for metrics |
+| service.internalMetrics.appProtocol | string | `""` | Adds the appProtocol field to the service. This allows to work with istio protocol selection. Ex: "http" or "tcp" |
+| service.internalMetrics.port | int | `8080` | internal metrics service port |
+| service.internalMetrics.portName | string | `"internal-metrics"` | name of the port for internal metrics. |
+| service.internalMetrics.targetPort | string | `nil` | targetPort overrides the internal metrics port. It defaults to the value of `internal_metrics.prometheus.port` from the Beyla configuration file. |
 | service.labels | object | `{}` | Service labels. |
 | service.loadBalancerClass | string | `""` | loadbalancer class name |
 | service.loadBalancerIP | string | `""` | loadbalancer IP |
 | service.loadBalancerSourceRanges | list | `[]` | source ranges for loadbalancer |
-| service.port | int | `80` | service port |
-| service.portName | string | `"metrics"` | name of the port for internal metrics. |
-| service.targetPort | int | `9090` | targetPort has to be configured based on the values of `BEYLA_INTERNAL_METRICS_PROMETHEUS_PORT` environment variable or the value of `prometheus_export.port` from beyla configuration file. see more at https://grafana.com/docs/beyla/latest/configure/options/#internal-metrics-reporter |
+| service.metrics.appProtocol | string | `""` | Adds the appProtocol field to the service. This allows to work with istio protocol selection. Ex: "http" or "tcp" |
```

Comment:
Moving `service` to `service.metrics` has all the sense but we would introduce a breaking change. What do the rest of the team thinks it's better?
1. Keep the breaking change, update major version for the chart and document it well.
2. Move back `service.metrics` to `service`, keeping also `service.internalMetrics`.

Reply:
I'd vote for option 2. I don't think breaking change here is worth it if we only want to rename some stuff.

Reply:
@mariomac @marctc I just committed a change that puts the Prometheus metrics port settings back on the `.service` level where it was before which removes the breaking change.